input/intro-notes/StructureDefinition-au-core-allergyintolerance-intro.md

```diff
@@ -22,3 +22,13 @@ Conformance in reverse is not guaranteed, i.e. a resource conforming to [Interna
 - The use of coding can vary significantly across systems, client applications need to understand that they may encounter codes they do not recognise and be prepared to handle those resources appropriately. Servers **SHOULD** populate `AllergyIntolerance.code.text` and/or `AllergyIntolerance.code.coding.display` so that client applications can at least display the condition even if the client application does not recognise the code supplied.
 - Refutation is not expected to be handled except as above - an appropriate negation code in `AllergyIntolerance.code` and `AllergyIntolerance.verificationStatus` of "confirmed" or "unconfirmed"
 - Where only substance is known (e.g. 111088007 \|Latex\|) and not a statement of allergy or intolerance (e.g. 300916003 \|Allergy to latex\|), the substance is sent in `AllergyIntolerance.code`
+
+<p class="request-for-feedback">Specific feedback is sought on the proposal to remove Must Support from AllergyIntolerance.asserter element. It's expected that many systems might choose not to use this field,  opting to derive its value from the Recorder entity. There's some uncertainty about how widespread this approach is across different systems' information recording processes.<br/>Please comment on HL7 Jira <a href="https://jira.hl7.org/browse/FHIR-44699">FHIR-44699</a>.</p>
+
+<div class="request-for-feedback"><p>Specific feedback is sought on the AllergyIntolerance.code refined subset binding proposal, based on the feedback received by vendors. This proposal recommends to:<ul>
```

Comment:
Too many words, leave a summary sentence and remove the rest.

Reply:
Done